postgrest_py/utils.py

```diff
@@ -1,7 +1,16 @@
+from httpx import AsyncClient  # noqa: F401
+from httpx import Client as BaseClient  # noqa: F401
+
+
+class SyncClient(BaseClient):
+    def aclose(self) -> None:
+        self.close()
```

Comment:
Hm why do we need this?

Reply:
It is a small deficiency in how the automatic generation of the synchronous code works.

If we see in this line (https://github.com/leynier/postgrest-py/blob/fc16d8bec997b7c34e0d7341ee72197d27e31f05/postgrest_py/_async/client.py#L36) of the asynchronous code (original code) when it is translated to its synchronous version (https://github.com/leynier/postgrest-py/blob/fc16d8bec997b7c34e0d7341ee72197d27e31f05/postgrest_py/_sync/client.py#L36) it continues to use the synchronous name `aclose` for the method so one way around is to implement that method in the synchronous version of the `httpx` client.

On the other hand, it is also necessary to create a `SyncClient` class, as the automatic translation of asynchronous to synchronous code changes the prefixes `Async` to `Sync`.

Reply:
Ah understood